pkg/goDB/engine/query.go

```diff
@@ -392,23 +408,62 @@ func createWorkManager(dbPath string, iface string, tfirst, tlast int64, query *
 	return
 }
 
-func parseIfaceList(dbPath string, ifaceList string) ([]string, error) {
+func parseIfaceListWithCommaSeparatedString(lister types.InterfaceLister, dbPath string, ifaceList string) ([]string, error) {
```

Comment:
Based on the change of the `InterfaceLister` interface, you can now also omit the `dbPath` parameter of this function.